### PR TITLE
add implementation for NAPI_GenerateQuote and NAPI_VerifyQuote

### PR DESCRIPTION
**Pull request by syan10**
add a new native unittest for the APIs.

Signed-off-by: Yan, Shaopu <shaopu.yan@intel.com>

### CODE REVIEW

File: core/App/ehsm_napi.cpp [hunk 1]
```diff
@@ -1446,29 +1472,42 @@ char* NAPI_GenerateQuote(const char *challenge)
...
+    sgx_ql_qv_result_t verifyresult;
+    bool result = false;
+
+    ehsm_data_t quote;
+    memset(&quote, 0, sizeof(quote));
 
-    // TODO: Verify Quote
-    result = true;
+    string quote_str = base64_decode(quote_base64);
+    quote.datalen = quote_str.size();
```
Comment (yang8621) [inline core/App/ehsm_napi.cpp:1493]: quote_str may be NULL?

File: core/App/ehsm_napi.cpp [hunk 2]
```diff
@@ -1446,29 +1472,42 @@ char* NAPI_GenerateQuote(const char *challenge)
  *          }
  *      }
  */
-char* NAPI_VerifyQuote(const char *quote, const char *nonce){
+char* NAPI_VerifyQuote(const char *quote_base64, const char *nonce)
+{
```
Comment (yang8621) [inline core/App/ehsm_napi.cpp:1476]: where check quote is base64 encoding? in node.js or add to this function?

Comment (syan10) [reply core/App/ehsm_napi.cpp:1476]: yes, checked in js layer

File: core/App/ehsm_napi.cpp (hunk 1 above)
Comment (syan10) [reply core/App/ehsm_napi.cpp:1493]: fixed

**Review by yang8621 — APPROVED** [review]

**Pull request merged**